tensorflow_decision_forests/contrib/temporal_feature_processor/temporal_feature_processor/implementation/pandas/operators/tests/assign/test_data/different_index.py

```diff
@@ -0,0 +1,55 @@
+import pandas as pd
+
+INPUT_1 = pd.DataFrame({
```

Comment:
Those test data great because they look real. However, they are relatively large making it hard to figure where is the salient information. What about making them shorter (if possible), and adding a comment to highly the salient information. For example, in this test, the important part is the different index.